app/bingey-api/models/user.js

```diff
@@ -0,0 +1,52 @@
+const { Schema, model } = require('mongoose');
+
+const userSchema = new Schema({
+    _id:{
+        type: String,
+        required: true,
+    },
+    name: {
+        type: String,
+        required: true,
+    },
+    password: {
+        type: String,
+        required: true,
+    },
+    registrationDate: {
+        type: Date,
+        required: true,
+    },
+    watchlists: [
+        {
+            watchlistId: {
+                type: Schema.Types.ObjectId,
+                ref: 'Watchlist',
+            },
+        }
+    ],
+    
+	recommended: [
+		{
+            titleId: {
+                type: Schema.Types.ObjectId,
+                ref: 'Titles',
+            },
+            name: {
+                type: String,
+            },
```

Comment:
I think keeping  such field declarations ( _fieldName: type_ ) shorter would be better
```suggestion
            name: String,
```